snapshot/lib/snapshot/detect_values.rb

```diff
@@ -47,6 +47,7 @@ def self.set_additional_default_values
 
           config[:devices] << sim.name
         end
+        config[:devices] << "Mac"
```

Comment:
Let's wrap the loop over simulators in `unless Snapshot.project.mac_app?` maybe?